front/app/api/seats/types.ts

```diff
@@ -2,8 +2,9 @@ import { Keys } from 'utils/cl-react-query/types';
 import seatsKeys from './keys';
 
 export type SeatsKeys = Keys<typeof seatsKeys>;
+export type TSeatType = 'collaborator' | 'admin';
```

Comment:
Is there a reason we want to move this here. It is only a concept in the SeatInfo and not the api or anywhere else.

Reply:
Good point. It can stay in SeatInfo indeed.